Doc/library/email.utils.rst

```diff
@@ -11,7 +11,7 @@
 There are a couple of useful utilities provided in the :mod:`email.utils`
 module:
 
-.. function:: localtime(dt=None)
+.. function:: localtime(dt=None, isdst=-1)
```

Comment:
Since it wasn't documented before and is being removed, we probably don't want to add it now :).  The deprecation note can mention that it was previously undocumented.